site/usage/applications.md

```diff
@@ -35,9 +36,15 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 -   [Vegas](https://github.com/aishfenton/Vegas) brings visualizations to Scala and Spark using Vega-Lite.
 -   [Vizard](https://github.com/yieldbot/vizard) tiny REPL client to visualize Clojure data in browser w/ Vega-Lite.
 -   [oz](https://github.com/metasoarous/oz) Vega & Vega-Lite based visualization toolkit for Clojure & ClojureScript (fork of Vizard).
+-   [Saite](https://github.com/jsa-aerial/saite) Exploratory graphics and ad hoc visualization system. Built in Clojure(Script) on top of [Hanami](https://github.com/jsa-aerial/hanami).
 -   [to-vega](https://github.com/gjmcn/to-vega) is a simple JavaScript library for creating Vega-Lite specifications.
 
 
+## Frameworks and Libraries utilizing Vega-Lite
+
+-   [Hanami](https://github.com/jsa-aerial/hanami)  A Clojure(Script) library and framework for creating domain and task specific interactive visualization applications using Vega-Lite and Vega. Built with [reagent](http://reagent-project.github.io/) (react) and [re-com](https://github.com/Day8/re-com) enabled.
```

Comment:
Do you think this may fit under "Tools for Embedding Vega-Lite Visualizations"? If not, please put this under "Tools that use Vega-Lite". I don't see that this needs a separate heading.

Reply:
I don't think it fits under "Tooils for Embedding ...", but as I mentioned in the thread,  it could fit under "Tools that use ..." as the closest existing heading.

The reason for having a heading like that is that most people don't think of libraries (or frameworks) as 'tools'.

Reply:
Let's put it under there and we can maybe adjust the heading as well.